core/modules/filter/filter.admin.inc

```diff
@@ -617,3 +617,105 @@ function filter_editor_image_upload_settings_form($format) {
 
   return $form;
 }
+
+
+/**
+ * Subform constructor to configure the text editor's file upload settings.
+ *
+ * Each text editor plugin that is configured to offer the ability to insert
```

Comment:
The comments should probably be more relevant to files and not just images.

Reply:
@herbdool yes, I was unsure what more to do here because this is an edited copy of the code for the image upload settings form, function filter_editor_image_upload_settings_form($format) at line 515, and it is able to re-use the behavior editorImageDialogue even though it is used here for files other than images. These new lines produce the right admin settings for files, as an addition to the settings separately produced for images, and these comments are specifically for this new admin section.